system-addon/lib/ASRouterTargeting.jsm

```diff
@@ -25,14 +26,21 @@ const TargetingGetters = {
   },
   get hasFxAccount() {
     return Services.prefs.prefHasUserValue(FXA_USERNAME_PREF);
+  },
+  get isInExperimentCohort() {
+    return Services.prefs.getIntPref(ONBOARDING_EXPERIMENT_PREF, 0);
   }
 };
 
+function EnvironmentTargeting(target) {
+  return {isFirstRun: target.url === "about:welcome"};
```

Comment:
The only thing that's unclear to me here is that doing it this way means that this experiment is dependent on the `about:welcome` experiment. That means that some of the users in the `about:welcome` cohort will be in these cohorts. I'm assuming that's fine? Forever confused about how these experiments are supposed to run together.....